Makefile

```diff
@@ -29,6 +29,8 @@ stan:
 test:
 	vendor/bin/sail composer test
 
+ci: lint stan test
```

Comment:
I think this must also be added to the `.PHONY:` line at the top?
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html